cmd/minishift/cmd/config/config.go

```diff
@@ -109,11 +108,62 @@ var settings []Setting = []Setting{
 		set:  SetString,
 	},
 	{
-		name: config.WantUpdateNotification,
+		name:        "host-config-dir",
+		set:         SetString,
+		validations: []setFn{IsValidPath},
+	},
+	{
+		name:        "host-data-dir",
+		set:         SetString,
+		validations: []setFn{IsValidPath},
+	},
+	{
+		name:        "host-only-cidr",
+		set:         SetString,
+		validations: []setFn{IsValidPath},
+	},
+	{
+		name:        "host-pv-dir",
+		set:         SetString,
+		validations: []setFn{IsValidPath},
+	},
+	{
+		name:        "host-volumes-dir",
+		set:         SetString,
+		validations: []setFn{IsValidPath},
+	},
+	{
+		name:        "http-proxy",
+		set:         SetString,
+		validations: []setFn{IsValidProxy},
+	},
+	{
+		name:        "https-proxy",
+		set:         SetString,
+		validations: []setFn{IsValidProxy},
+	},
+	{
+		name: "insecure-registry",
```

Comment:
@hferentschik @LalatenduMohanty @budhrg @gbraad This is what I added and set a Slice method.

Now if I do like below then it does put value of insecure registry as slice but it neglected by minishift start.

```
$ ./minishift config set insecure-registry 17.0.0.1/32,63.0.0.1/23

17:40 $ ./minishift config view
- insecure-registry    : [17.0.0.1/32 63.0.0.1/23]
- show-libmachine-logs : true
```